### PR TITLE
Rails v5 support

### PR DESCRIPTION
**Pull request by pboling**
Firstly, and most importantly, I have no expectation you'd want to add support for Rails 5.2.

I need to use this gem across all my Rails apps, which span 5.2 to 7.2, so I can unify my logging.  I've just released a backport of the Rails 8 beta's logger and tagged logger, working on Ruby 2.7 and Rails 5.2, to that end.

- https://github.com/pboling/activesupport-logger
- https://github.com/pboling/activesupport-tagged_logging

I expect there are many others in the same boat who want JSON logs everywhere, so making this PR for Google visibility.

If this isn't accepted, and if I don't release a forked gem, then for now:
```
gem "json_tagged_logger", "~> 0.7", github: "pboling/json_tagged_logger", branch: "rails-v5-support"
```

### CODE REVIEW

**Comment by santry** [comment]
Hey @pboling thanks for sending this along. I'm not opposed to adding Rails 5.2 support but it looks like there's a bunch of other stuff here that I don't think is necessarily appropriate for adding to the gem. Does PR #19 satisfy the basic need to have a gem that works in 5.2? It's passing CI but I don't have an actual 5.2 (or 6.0) app to test in at the moment.

File: lib/json_tagged_logger/formatter.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def call(severity, _time, _progname, message)
         log[:tags] = text_tags.to_a
       end
 
-      bare_message = message_without_tags(message)
```
Comment (pboling) [inline lib/json_tagged_logger/formatter.rb:25]: @santry I think this is the only thing missing, but it is actually for compatibility with Rails 8, where sending `nil` no longer works, it expects a string.

File: lib/json_tagged_logger/formatter.rb [hunk 2]
```diff
@@ -22,7 +22,7 @@ def call(severity, _time, _progname, message)
...
+      bare_message = message_without_tags(message.to_s)
```
Comment (pboling) [inline lib/json_tagged_logger/formatter.rb:25]: @santry I think this (adding `.to_s`) is the only thing missing from #19, but it is actually for compatibility with Rails 8, where sending `nil` no longer works, it expects a string.

Comment (santry) [reply lib/json_tagged_logger/formatter.rb:25]: Yep, I added that in the already-merged #18.

**Comment by pboling** [comment]
Closing in favor of #18 and #19

**Pull request closed without merging**